### DISCUSSION TITLE
Integration with Bitbucket

### DISCUSSION THREAD

**Discussion by ankitashome** [datreeio/datree#107] [Q&A]
Is it possible to integrate Datree with hosted Bitbucket, so that the rules/policy validations can be hooked to the git repos and can be a step that runs as soon as any commit is pushed, making it fail fast and correct fast too?

**Comment by eyarz** [comment]
If I understand correctly, you're asking if we have a [BitBucket app](https://marketplace.atlassian.com/addons/app/bitbucket)?
We don't have something like that (yet), but you can still use Datree as a pre-commit/push hook or in your CI :)
If you need help with one of those options - let me know.
